js/views/transactionsVw.js

```
@@ -281,6 +285,118 @@ module.exports = baseVw.extend({
+      dataObject.refund_address = '';
+      dataObject.moderator_name = '';
+      dataObject.moderator_guid = '';
+      dataObject.moderator_fee = '';
+      dataObject.return_policy = '';
+
+      //format and add flat data to the object
+      if(data.buyer_order){
+        var dPayment = data.buyer_order.order.payment;
+
+        dataObject.quantity = data.buyer_order.order.quantity;
+        if(data.buyer_order.order.shipping){
+          var dShipping = data.buyer_order.order.shipping;
+          dataObject.shipping_address = dShipping.ship_to + " " + dShipping.address +", " + dShipping.city + ", " + dShipping.state + ", " + dShipping.postal_code + ", " + dShipping.country;
+        }
+        dataObject.payment_amount = dPayment.amount;
+        dataObject.payment_address = dPayment.address;
+        dataObject.refund_tx_fee = dPayment.refund_tx_fee || "";
+        dataObject.chaincode = dPayment.chaincode || "";
+        dataObject.redeem_script = dPayment.redeem_script || "";
+        dataObject.refund_address = data.buyer_order.order.refund_address;
+
+        if (data.buyer_order.order.moderator) {
+          var matchedModerator = data.vendor_offer.listing.moderators.filter(function (moderator) {
+            return moderator.guid == data.buyer_order.order.moderator;
+          });
+          dataObject.moderator_name = matchedModerator[0].name;
+          dataObject.moderator_guid = matchedModerator[0].guid;
+          dataObject.moderator_fee = matchedModerator[0].fee;
+        }
+
+      }
+
+      if(data.vendor_offer.policy){
+        dataObject.return_policy = data.vendor_offer.policy.returns;
+      }
+
+      dataObject.raw_contract_data = JSON.stringify(data, null, 2);
+
+    }).always((data, textStatus)=> {
+      if(textStatus == 'parsererror'){
+        alert(window.polyglot.t('errorMessages.serverError'), window.polyglot.t('errorMessages.badJSON'));
+      }
+      this.currentExportData.push(dataObject);
```

That was intentional, so that the passed in dataObject gets passed back with the non-AJAX changes if the AJAX fails. 

Though that raises the question of whether it's better to stop the process if a fail happens rather than return partial data. It's probably better to stop it.